build.gradle

```diff
@@ -14,10 +14,10 @@ plugins {
     id 'pmd'
     id 'com.github.ben-manes.versions' version '0.27.0'
     id "info.solidsoft.pitest" version '1.3.0'
-    id 'io.spring.dependency-management' version '1.0.8.RELEASE'
+    id 'io.spring.dependency-management' version '1.0.9.RELEASE'
     id 'org.owasp.dependencycheck' version '5.2.2'
     id 'org.sonarqube' version '2.8'
-    id 'org.springframework.boot' version '2.2.2.RELEASE'
+    id 'org.springframework.boot' version '2.2.4.RELEASE'
     id "org.flywaydb.flyway" version "6.1.0"
```

Comment:
there are couple more changes in PRD build.gradle file regarding dependency versions. Please check.

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-user-profile-api/pull/240/commits/9a800a1c26dc64dead6d101103e08710580fc129